### DISCUSSION TITLE
[Technical Questions] Failure in push origin

### DISCUSSION THREAD

**Discussion by EvelynBunnyDev** [CS342/.github#62] [Technical Questions] [answered]
### In what area do you have a technical challenge?

Other (non-StanfordBDHG-related) area

### Description

I encountered an error while pushing my local changes to github. Screenshots attached. 
After clicking push origin, errors occurred. I did not have this error in previous pushes.
<img width="640" alt="Screenshot 2024-02-06 at 1 10 48 PM" src="https://github.com/CS342/.github/assets/153574461/fdcfaec9-a2ba-42c7-a8fc-ae2c9dfe8802">
<img width="747" alt="Screenshot 2024-02-06 at 1 10 54 PM" src="https://github.com/CS342/.github/assets/153574461/60d0244b-b5f3-45b0-9d57-9005b09d0c2d">

### Reproduction

NA

### Expected behavior

NA

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by EvelynBunnyDev** [comment, marked as answer]
I just realized that I did not create another branch for this!! Silly question

**Reply by PSchmiedmayer** [reply]
Thank you for asking the question here and sharing your solution @EvelynBunnyDev! Sometimes happens to me as well.
